ivy/functional/backends/torch/converters.py

```diff
@@ -28,13 +28,13 @@ def __init__(
         )
         ivy.Module.__init__(self, device=device, devices=devices)
 
-    def _create_variables(self, device):
+    def _create_variables(self, device, dtype):
         return self._native_params
 
-    def _build(self):
+    def _build(self, *args, **kwargs):
         self._native_module = ivy.default(
             lambda: self._native_module,
-            lambda: self._native_module_class(*self._args, **self._kwargs),
+            lambda: self._native_module_class(*args, **kwargs),
```

Comment:
For `torch's Module` class  , there is no `dtype` argument, so this line will raise an error. https://pytorch.org/docs/stable/_modules/torch/nn/modules/module.html#Module. Also, I would suggest running the `test_converteres` to look for any more errors.

Reply:
@djl11 As of now this is not causing issues with `ivy.Module` but it is causing issues with `ivy.converters` .

Reply:
@RickSanchezStoic Thanks. As I ran the `test_converters.py`, there were no failing tests (4 skipped ones). How may I check the potential errors?